src/locales/fi.json

```diff
@@ -682,7 +682,19 @@
     "noticeDetails": {
       "title": "Nostojärjestysilmoitus",
       "notice": "Ilmoitus",
-      "ownerInformation": "Omistajan tiedot"
+      "area": "Alue",
+      "ownerInformation": "Omistajan tiedot",
+      "stickerDetails": {
+        "title": "TARRA",
+        "number": "Numero",
+        "season": "Kausi",
+        "assign": {
+          "buttonText": "Vaihda numero",
+          "modalTitle": "VAIHDA NUMERO",
```

Comment:
I would prefer naming these by content and using `.toUpperCase()` where needed: `"sticker": "Tarra"`, `"changeNumber": "Vaihda numero"`

Reply:
I've meant to name these translation keys by where they appear on the UI. Not sure how to best place them, and this issue also relates to many other translations. There is duplication currently in many places, and the uppercasing could indeed be done in code: either js or css.

Should probably create a new task for refactoring the whole translation file structure, but the benefits of such a refactor will only become relevant once we need translations for multiple languages.